docs/components/integration-framework/connectors/out-of-the-box-connectors/googledrive.md

```diff
@@ -0,0 +1,200 @@
+---
+id: googledrive
+title: Google Drive connector
+description: Create empty folders or documents from templates from your BPMN process.
+---
+
+The **Google Drive connector** allows you to create empty folders or documents from templates from your BPMN process.
+
+## Prerequisites
+
+To start working with the **Google Drive connector**, a relevant OAuth token must be configured and stored as a secret in the user's cluster. The token has to have permission to read and create a folder and/or files from a desired Google Drive instance. Follow the steps from the appendix to find out more about creating an OAuth token and giving relevant permissions.
+
+## Create a Google Drive connector task
+
+Currently, the **Google Drive connector** supports two flavors: create a new folder and create a file from a template.
+
+To use a **Google Drive connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu or create a new connector task by using the **Append connector** context menu. Follow our guide on using connectors to learn more.
```

Comment:
When you say "Follow our guide on using connectors to learn more" -- please link to the connectors guide.

Reply:
Updated